### ISSUE TITLE
Extension doesn't work

### ISSUE THREAD

**Issue by NicholasPilotto** [microsoft/vscode-cpptools#5183]
**Type: LanguageService**

**Describe the bug**
- OS and Version: macOS Catalina 10.15.3
- C/C++ Extension Version: 0.27.0
- Visual Studio Code Version: 1.43 (February 2020)
- Bug description: the entire extension doesn't work, the autocomplete code doesn't appear, the formattingOnSave doesn't format

**Expected behavior**

Before the last update, everything was working fine, but now nothing happens. When you type some code the autocomplete window doesn't appear, never. If is set up the autoformatting on save, when you save, nothing happens

Log: 
- cannot activate extension: 'ms-vscode.cpptools’
- ERR Cannot read property 'ActiveClient' of undefined: TypeError: Cannot read property ActiveClient' of undefined at Object. .<anonymous> /Users/<User>/.vscode/extensions/ms-vscode.cpptools-0.27.0-insiders3/dist/main.js:4284:33

**Comment by sean-mcmanus** [comment]
Are you using 0.27.0-insiders4? Can you check the Toggle Developer Tools logs to see if our extension is crashing on startup?

**Comment by NicholasPilotto** [comment]
I checked the Toggle Developer Tools log and give me an error: “cannot activate extension: 'ms-vscode.cpptools’” and the reason "Empty URI in client constructor”, and some errors like 
"ERR Cannot read property 'ActiveClient' of undefined: TypeError: Cannot read property 'ActiveClient' of undefined
    at Object.<anonymous> (/Users/NicholasPilotto/.vscode/extensions/ms-vscode.cpptools-0.27.0-insiders3/dist/main.js:4284:33)"

Is it a bug or I have to do something? Like update or maybe downgrade the extension.
Thanks for your attention.

P.S.: I checked the VS Code Version: February 1.43.2

Nicholas Pilotto
